ikasaneip/component/endpoint/amazon-s3-endpoint/src/main/java/org/ikasan/component/endpoint/amazon/s3/client/TransferManagerProgressLogger.java

```diff
@@ -0,0 +1,51 @@
+package org.ikasan.component.endpoint.amazon.s3.client;
+
+import com.amazonaws.services.s3.transfer.Transfer;
+import com.amazonaws.services.s3.transfer.TransferProgress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Used to show a progress bar of the upload in the log
+ */
+public class TransferManagerProgressLogger {
```

Comment:
I am not sure of the usefulness of that in application log

Reply:
This is actually quite useful when loading a large 2GB file as you can see how far it is along - for small files it will just print a full bar. My personal opinion is when there is a long running process this should be indicated in the log otherwise the user has no way of determining if processing has "hung"

Reply:
Changed to % rather than progress bar as discussed